benchmarks/project/README.md

```diff
@@ -4,16 +4,11 @@ This project contains code for the starter and worker, which are used during our
 
 ## Build docker images for benchmark application
 
-### Update the Zeebe version (only required once a release cycle)
+To build the docker images for the benchmark application, run the following command:
 
-```
-mvn versions:set-property -DgenerateBackupPoms=false -Dproperty=version.zeebe -DnewVersion=X.Y.Z-SNAPSHOT
-```
-
-### Build and push the images
-
-```
-docker-compose build
-docker-compose push
+```bash
+./mvnw -am -pl benchmarks/project package -DskipTests -DskipChecks
+./mvnw -pl benchmarks/project jib:build -Pstarter
```

Comment:
:rocket:

Reply:
Be aware that we might need to change the release process and mention this as well, right? Or is this now automatically done on release? I think you haven't changed that ?

Reply:
Right, let me check where we mention this in the release process

Reply:
The release process only [mentions](https://github.com/zeebe-io/zeebe-engineering-processes/blob/f6d4acb4e4d4b996c6b7b7f06ffbed0caf909c4d/src/main/resources/release/post_release.bpmn#L108) the `release.sh` script here, which I've adjusted.